src/text-expander-element.ts

```diff
@@ -2,22 +2,30 @@ import Combobox from '@github/combobox-nav'
 import query from './query'
 import {InputRange} from 'dom-input-range'
 
-type Match = {
+export type TextExpanderMatch = {
   text: string
   key: string
   position: number
 }
 
-type Result = {
+export type TextExpanderResult = {
   fragment: HTMLElement
```

Comment:
`fragment` could be optional if `matched=false`?

Reply:
Yes it could, as `fragment` is not accessed when `matched` is false:

https://github.com/github/text-expander-element/blob/954090fd1c53ca66e238894d85e8041b73cdab4d/src/text-expander-element.ts#L248

Reply:
Done.